<launch>
  <node ns="binaural_audio" pkg="binaural_microphone" name="webcam_stereo" type="webcam_stereo.py" output="screen" />
</launch>
